packages/material-ui-lab/src/TimelineItemDot/TimelineItemDot.js

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { capitalize } from '@material-ui/core/utils';
+import { withStyles } from '@material-ui/core/styles';
+
+export const styles = (theme) => ({
+  /* Styles applied to the root element. */
+  root: {
+    display: 'flex',
+    alignSelf: 'baseline',
+    borderStyle: 'solid',
+    borderWidth: 2,
+    flex: 0,
+    padding: 4,
+    borderRadius: 999,
+    // marginRight: 'auto',
+    // marginLeft: 'auto',
+    top: 8,
+    position: 'relative',
+    boxShadow: theme.shadows[2],
+  },
+  /* Styles applied to the root element if `color="default"` and `variant="default"`. */
+  defaultDefault: {
+    borderColor: theme.palette.grey[300],
+    backgroundColor: theme.palette.grey[300],
+  },
+  /* Styles applied to the root element if `color="default"` and `variant="outlined"`. */
```

Comment:
We might want to add background color on the dot. We might also have a tail alignment issue (top vs bottom)

```suggestion
  outlined: {
     backgroundColor: theme.palette.background.paper,
   },
  /* Styles applied to the root element if `color="default"` and `variant="outlined"`. */
```

<img width="160" alt="outlined" src="https://user-images.githubusercontent.com/3165635/83942985-c4944200-a7f8-11ea-9845-406c7824a317.png">

Reply:
See comment: https://github.com/mui-org/material-ui/pull/21331#discussion_r436339352